packages/bruno-common/src/example-status/index.ts

```
@@ -0,0 +1,41 @@
+          }
+        });
+      }
+
+      if (item.items && item.items.length) {
+        transformItems(item.items);
+      }
+    });
+  };
+
+  if (Array.isArray(collection)) {
+    collection.forEach((col) => transformItems(col.items));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)